Remotion/Web/Development.WebTesting.TestSite.Shared/ImageTest.aspx

```diff
@@ -14,13 +14,14 @@
  % You should have received a copy of the GNU Lesser General Public License
  % along with re-motion; if not, see http://www.gnu.org/licenses.
 --%>
-<%@ Page Language="C#" MasterPageFile="~/Layout.Master" AutoEventWireup="true" CodeBehind="ImageTest.aspx.cs" Inherits="Remotion.Web.Development.WebTesting.TestSite.ImageTest" %>
+<%@ Page Language="C#" MasterPageFile="Layout.Master" AutoEventWireup="true" CodeBehind="ImageTest.aspx.cs" Inherits="Remotion.Web.Development.WebTesting.TestSite.Shared.ImageTest" %>
+<%@ Import Namespace="Remotion.Web.Development.WebTesting.TestSite.Shared" %>
 <asp:Content ContentPlaceHolderID="body" runat="server">
   <h3>Image</h3>
-  <img id="MyImage" src="~/Images/SampleIcon.gif" alt="My alternative text" runat="server"/>
+  <img id="MyImage" src="data:image/gif;base64,R0lGODlhEAAQAPH/AAAAAAAAgMDAwP///yH5BAUAAAQALAAAAAAQABAAAANESLrcriBKEoIDImtQG8ZaVkmR8IXCOAznmXEBwJrzF1ExK68fu+SrIIvH2AmFgKJOJ/EtjEfic0mSJafR4BVS7T6+jQQAOw==" alt="My alternative text" runat="server"/>
```

Comment:
I don't think those images actually need to runat-server? Would simply switch this to <%=ResolveUrl / Resource > (depending on what works best.